pyproject.toml

```diff
@@ -2,54 +2,53 @@
 requires = [
   "setuptools>=62.1",
   "setuptools_scm[toml]>=6.2",
-  "wheel",]
+  "wheel",
+]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "sunkit_magex"
 description = "Solar Magnetic field Extrapolation"
 requires-python = ">=3.10"
 readme = { file = "README.rst", content-type = "text/x-rst" }
-license = { file = "licenses/LICENSE.rst", content-type = "text/plain" }
```

Comment:
If we want to change this we should do it in the template, although I don't see the issue.

Reply:
Fixed upstream